index.njk

```diff
@@ -2,42 +2,69 @@
 layout: layouts/base.njk
 ---
 
-<div class="container">
+{# <div class="container">

```

Comment:
Is this commented out? 🤔

Reply:
Yeah. Should I just delete those 3 lines at the top? I am no longer going to need them anyways